plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -464,6 +464,12 @@ def make_template(in_str):
                     node=node,
                     manifest=self.dbt_manifest,
                 )
+            except Exception as err:
+                # Additional error logging in case we get a fatal dbt error.
+                raise SQLTemplaterSkipFile(  # pragma: no cover
```

Comment:
What does the logging look like from this? Curious if we should do one or both of:

Logging where the exception was raised from:
```
logger.exception("<<MESSAGE>>")
```

Exception chaining:
```
raise SQLTemplaterSkipFile(...) from err
```

Reply:
Both are things I've never used before 🤔 .

On reading - I think we should definitely do the latter. I think it will still result in a skipped file. Doing the former seems like a good idea too. Perhaps with some more information attached.

Great ideas!

Reply:
I'm a bit of a Python 🤓, I suppose. 😝

Reply:
Take a look at what I've done now. Used both 🤓

Reply:
Would this resolve a problem I have seen where sqlfluff appears to have worked when in fact dbt didnt compile?

![image](https://user-images.githubusercontent.com/19945/179065275-052e573e-41cc-4c7f-b2c0-f2d2b6728f75.png)